README.md

```diff
@@ -19,13 +19,13 @@ The greatest difficulties encountered right now are caused by these issues that
  - Lack of ground truth (No True 4K Anime)
  - Few true 1080p anime (Even some anime mastered at 1080p have sprites that were downsampled)
  - Non-1080p anime are upsampled to 1080p using simple algorithms, resulting in a blurry 1080p image. Our algorithm has to detect this. (Main reason why waifu2x does not work well on anime)
- - UV channels of anime are subsampled (4:2:0), which means the color channels of 1080p anime are in fact 540p, thus there is a lack of 1080p ground truth for the UV channels.
+ - UV channels of anime are subsampled (4:2:0), which means the color channels of 1080p anime are 540p, thus there is a lack of 1080p ground truth for the UV channels.
  - Simulating H.264/H.265 compression artifacts (for analysis and denoising) is not trivial and is relatively time-consuming.
- - Due to the workflow of animation studios and their lack of time/budget, resampling artifacts of individual sprites are present in many modern anime.
- - Speed (preferably real-time) is paramount, since we do not want to re-encode video each time the algorithm improves. There is also less risk of permanently altering original content.
+ - Due to the workflow of animation studios and their lack of time/budget, resampling artifacts of individual sprites are present in much modern anime.
+ - Speed (preferably real-time) is paramount since we do not want to re-encode video each time the algorithm improves. There is also less risk of permanently altering the original content.
```

Comment:
Line 24: "Much modern anime" is not correct. Rather, "A sizeable portion of modern anime" or similar sentences seem more legible imo.